acceptance/managers/input_configuration_manager.go

```diff
@@ -0,0 +1,91 @@
+// +build acceptance
+
+package managers
+
+import (
+	"encoding/json"
+	"os"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+)
+
+const (
+	envPackPath                 = "PACK_PATH"
```

Comment:
The only ones of these I see used is GITHUB_TOKEN and ACCEPTANCE_SUITE_CONFIG. Where are the rest used? Can we take them out?

Reply:
These are used in [`NewInputConfigurationManager`](https://github.com/buildpacks/pack/pull/708/files#diff-638d0c574e1c03af8d170c505421b931R36-R42) unless I'm missing your point?

Reply:
What I had meant is, are those actually set by any environments/users such that the values in `NewInputConfigurationManager` will ever not be `""`? It's great that we offer this flexibility, I just didn't see anywhere that actually set that environment variable, though I easily could have missed it.

Reply:
Yeah, I think it's pretty important to maintain our older contract (prior to automatically downloading missing pre-requisites these variables were required). Auto-downloading should be considered a helper and a fallback to defining these variables.